packages/widgets/src/blocks/legacy-widget/editor.scss

```diff
@@ -67,6 +67,8 @@
 			line-height: 1;
 			min-height: 30px;
 			padding-left: $grid-unit-10;
+			padding-top: $grid-unit * 1.25;
+			padding-bottom: $grid-unit * 1.25;
```

Comment:
Why `10px`? Maybe we can do `8px` (`$grid-unit`) too?

Reply:
Because that value (10px) is being used on the Widgets page.
In my opinion, we should use similar values for the same component.
But if you insist I can change it to 8px.

Reply:
Could you point me to the place it's used on the Widgets page? 🤔

Reply:
It's on Appearance -> Widgets page. Sorry if it wasn't clear.
Just add a new (legacy) Navigation Menu widget.
![Screenshot 2021-08-17 at 9 52 33](https://user-images.githubusercontent.com/43744263/129686661-84b55df6-3b40-46e6-8371-e2a63582839f.png)

Reply:
Correct me if I'm wrong, but I think it's defined by the theme (`twentytwentyone`). You might see other values if switch to other themes.

Reply:
Yep, it is. 8px vs 10px is not such a big difference.
I think we can use 8px as well.